src/legend.ts

```diff
@@ -88,12 +89,12 @@ export interface Legend extends BaseLegendNoSignals, LegendMixins, Guide {
    *
    * __Default value__: `undefined`
    */
-  tickMinStep?: number;
+  tickMinStep?: number | SignalRef;
```

Comment:
This is also a Vega property, right? Why do we need to override here?

Reply:
Same issue with axis, let's handle later.